packages/terra-form-select/tests/wdio/native-select-spec.js

```diff
@@ -0,0 +1,176 @@
+Terra.describeViewports('Native Select', ['tiny'], () => {
+  describe('Default', () => {
+    browser.url('/raw/tests/terra-form-select/form-select/native-select/default');
+
+    Terra.it.validatesElement('initial');
+
+    it('should apply hover to the select', () => {
+      browser.moveToObject('#select-id', 5, 5);
+    });
+
+    Terra.it.validatesElement('hover');
+
+    it('should set focus to the select', () => {
+      browser.click('#root', 0, 0);
+      browser.keys('Tab');
+    });
+
+    Terra.it.validatesElement('focus');
+
+    it('should apply hover and focus to the select', () => {
+      browser.moveToObject('#select-id', 5, 5);
+    });
+
+    Terra.it.validatesElement('hover-and-focus');
+  });
+
+  describe('Disabled', () => {
+    before(() => browser.url('/raw/tests/terra-form-select/form-select/native-select/disabled'));
+
+    Terra.it.validatesElement('initial');
+
+    it('should apply hover to the select', () => {
+      browser.moveToObject('#select-id', 5, 5);
+    });
+
+    Terra.it.validatesElement('hover');
+
+    it('should set focus to the select', () => {
```

Comment:
I think this should be `'should not set focus to the select'`

Reply:
Updated.